pymc3/distributions/transforms.py

```diff
@@ -208,12 +219,19 @@ class Interval(ElemwiseTransform):
     def __init__(self, a, b):
         self.a = tt.as_tensor_variable(a)
         self.b = tt.as_tensor_variable(b)
+        self.a_ = a
+        self.b_ = b
 
     def backward(self, x):
         a, b = self.a, self.b
         r = (b - a) * tt.nnet.sigmoid(x) + a
         return r
 
+    def backward_val(self, x):
+        a, b = self.a_, self.b_
+        r = (b - a) * 1/(1+np.exp(-x)) + a  ## fix this
```

Comment:
spaces around mathematical operators. does it still need to be fixed?

Reply:
that comment is about something different, haha, we have to remove it! can we use black to format this? Or we should use autopep8?

Reply:
black for new files works well I think.